apps/ledger-live-mobile/src/screens/OperationDetails/Content.tsx

```diff
@@ -387,7 +387,7 @@ export default function Content({
                   showCode
                   disableRounding={true}
                   date={operation.date}
-                  subMagnitude={1}
+                  subMagnitude={5}
```

Comment:
Can you please explain this change? Thanks

Note that because this file isn't specific to Icon the change would affect all other integrations, and me way require to implement it differently.